examples/src/main/java/org/apache/spark/examples/streaming/JavaTwitterHashTagJoinSentiments.java

```diff
@@ -171,5 +171,6 @@ public void call(JavaPairRDD<Double, String> happinessTopicPairs) {
 
     jssc.start();
     jssc.awaitTermination();
+    jssc.close();
```

Comment:
Ah, I just realized I misled you in my comments. For _streaming_ examples, we should _not_ call `close()` or `stop()`. The correct last call is `awaitTermination()`. If you back those out, I'll merge this. Sorry about that.